python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1125,6 +1127,86 @@ async def move_m2_hexapod(
             component_name="M2 Hexapod",
         )
 
+    async def offset_camera_hexapod(
+        self,
+        x: float,
+        y: float,
+        z: float,
+        u: float,
+        v: float,
+        w: float = 0.0,
+        sync: bool = True,
+    ) -> None:
+        """Offset camera hexapod.
```

Comment:
I would make it a bit more explicit here, considering the compensation mode state. Maybe something like this? 
```
Apply an offset to the camera hexapod current position, 
regardless of the compensation mode status.
```